scripts/cpplinter.sh

```diff
@@ -16,6 +16,8 @@ EXCLUDED_PATTERNS="\
     ecc_enclave/_build \
     tlcc_enclave/nanopb \
     tlcc_enclave/_build \
+    examples/auction/_build \
```

Comment:
to keep this file simpler, maybe we could just exclude /_build instead of listing every single directory?